services/dub/dub-score.service.js

```diff
@@ -0,0 +1,42 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+import { colorScale } from '../color-formatters.js'
+
+const schema = Joi.object({
+  score: Joi.number().required(),
+})
+
+export default class DubScore extends BaseJsonService {
+  static category = 'rating'
+
+  static route = { base: 'dub/score', pattern: ':packageName' }
+
+  static examples = [
+    {
+      title: 'DUB SCORE',
+      namedParams: { packageName: 'vibe-d' },
+      staticPreview: this.render({ score: 4.5 }),
+    },
+  ]
+
+  static defaultBadgeData = { label: 'SCORE' }
+
+  static render({ score }) {
+    return {
+      label: 'SCORE',
```

Comment:
Labels should be lowercase (x2)

https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines